### PR TITLE
Feat/ideas list

### PR DESCRIPTION
**Pull request by leojclarke**

### CODE REVIEW

File: src/app/IdeasFeed.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import styled from 'styled-components'
+import IdeaPost from './IdeaPost'
+
+const IdeasList = styled.div`
+  padding: 10px 10px;
```
Comment (AnsMann) [inline src/app/IdeasFeed.js]: you can use 10px once

File: src/app/Tag.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const StyledTag = styled.span`
+  padding: 3px;
+  margin: 0px 5px 0px 0px;
```
Comment (AnsMann) [inline src/app/Tag.js]: you can use `margin-right: 5px`

File: src/app/Tag.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  background: rgba(127, 183, 190, 1);
```
Comment (AnsMann) [inline src/app/Tag.js]: you can use only `rgb()` because alpha = 1

File: src/app/Tag.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const StyledTag = styled.span`
```
Comment (AnsMann) [inline src/app/Tag.js]: you can think about sorting your components CSS in alphabetical order

File: src/app/Footer.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const StyledFooter = styled.div`
```
[the remark this answers is not in the document]
Comment (leojclarke) [reply src/app/Footer.js]: agreed, done!

File: src/app/Header.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const Heading = styled.div`
```
[the remark this answers is not in the document]
Comment (leojclarke) [reply src/app/Header.js]: Also agreed, and done...!

File: src/app/IdeaPost.js [hunk 7]
```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import styled from 'styled-components'
+import Tag from './Tag'
+
+const IdeaContainer = styled.div`
```
[the remark this answers is not in the document]
Comment (leojclarke) [reply src/app/IdeaPost.js]: done.

**Pull request merged**